compiler/src/iree/compiler/Codegen/Common/Transforms.cpp

```diff
@@ -0,0 +1,172 @@
+// Copyright 2024 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "iree/compiler/Codegen/Common/Transforms.h"
+#include "mlir/Dialect/Affine/IR/AffineOps.h"
+
+#define DEBUG_TYPE "iree-codegen-common-transforms"
+
+namespace mlir::iree_compiler {
+
+/// Pattern to convert `tensor.extract_slice(tensor.expand_shape)` to
+/// `tensor.expand_shape(tensor.extract_slice)`.
```

Comment:
I know it was wrong before the move. Please help update the comment. It is not a pattern since you're touching the code.